script.R

```diff
@@ -10,6 +10,53 @@ if_error <- function(x, y = NULL) {
 `%||%` <- function(x, y) {
   if (!length(x) || is.null(x)) y else x
 }
+check_if_pkg_available <- function(pkg, ver = NULL) {
```

Comment:
Another command about `check_if_pkg_available`
Maybe we can introduce `repos` parameter, that is passed to `available.packages` so it's clear we seek for packages within `repos`. and `repos` will be `getOptions('repos')` of course

Reply:
thanks
good comment but I think I would leave it as is to keep it more simpler